media-source-respec.html

```diff
@@ -369,6 +369,10 @@ <h3>Attributes</h3>
           <p>The event handler for the {{sourceended}} event.</p>
         </dd><dt><dfn><code>onsourceclose</code></dfn> of type {{EventHandler}}</dt><dd>
           <p>The event handler for the {{sourceclose}} event.</p>
+        </dd>
+        <dt><dfn><code>canConstructInDedicatedWorker</code></dfn> of type {{boolean}}</dt><dd>
+          <p>Returns true.</p>
+          <p class="note">This attribute enables feature detection of support for MSE-in-Workers without requiring an application to attempt usage first from a DedicatedWorker.</p>
```

Comment:
"MSE-in-Workers" and "DedicatedWorker" seem jargon to me. Suggest to expand with regular prose as in:

```suggestion
          <p class="note">This attribute enables feature detection of support for creating and using a {{MediaSource}} object in a dedicated worker directly from the main thread without requiring an application to attempt usage first from a dedicated worker.</p>
```

Reply:
Good point. 9efdf52 addresses this comment.